WORKSPACE

```diff
@@ -14,3 +14,9 @@ http_archive(
 load("//bzl:workspace.bzl", "plaidml_workspace")
 
 plaidml_workspace()
+
+new_local_repository(
+    name = "cmheaders",
+    build_file = "cmheaders.BUILD",
+    path = "/usr/include/",
```

Comment:
This won't work on our build machines or for public contributors. Is there a public URL that we can use to download the headers for CM?

Reply:
Yes, the cmheaders are contained in a zip file in https://01.org/c-for-metal-development-package/downloads. Is the size(>200MB) too big? Or I can include the headers in CM github repo (about 16MB) where the headers are not contained in a single folder. I have tried to add it in a http_archive in plaidml/bzl/workspace.bzl, but the cmheader files also depend on other header files in the /usr/include path. I will try to add all dependences from public URLs.